### ISSUE TITLE
error running game

### ISSUE THREAD

**Issue by ziv** [RDGT/adventure-poc#17]
from master branch, running game throws this error:
```
Traceback (most recent call last):
  File "adventure.py", line 4, in <module>
    import game_code
  File "/home/ziv/code/adventure-poc/game/game_code/__init__.py", line 1, in <module>
    import main
  File "/home/ziv/code/adventure-poc/game/game_code/main.py", line 7, in <module>
    import asset_loader
  File "/home/ziv/code/adventure-poc/game/game_code/asset_loader/__init__.py", line 1, in <module>
    from json_parser import JsonParser
  File "/home/ziv/code/adventure-poc/game/game_code/asset_loader/json_parser.py", line 4, in <module>
    from core import exceptions, assets_dir
ImportError: No module named core
```

**Comment by InbarRose** [comment]
Thank you.
This bug was introduced when merging test-adventure.py changes

fix will require changing the imports to assume CWD/source root is project root and not assume that the context includes game_code

**Comment by InbarRose** [comment]
fixed in commit 4d0d554
